rules/tacacs.mk

```diff
@@ -29,6 +29,19 @@ $(LIBNSS_TACPLUS)_RDEPENDS += $(LIBTAC2)
 $(LIBNSS_TACPLUS)_SRC_PATH = $(SRC_PATH)/tacacs/nss
 SONIC_MAKE_DEBS += $(LIBNSS_TACPLUS)
 
+
+# libnss-tacplus packages
+BASH_TACPLUS_VERSION = 1.0.0
+
+export BASH_TACPLUS_VERSION
+
+BASH_TACPLUS = libnss-tacplus_$(BASH_TACPLUS_VERSION)_$(CONFIGURED_ARCH).deb
+$(BASH_TACPLUS)_DEPENDS += $(LIBTAC_DEV)
+$(BASH_TACPLUS)_RDEPENDS += $(LIBTAC2)
+$(BASH_TACPLUS)_SRC_PATH = $(SRC_PATH)/tacacs/bash
```

Comment:
>bash [](http://example.com/codeflow?start=46&length=4)

Is it possible to remove one level `bash` in the path? slave.mk support build a deb from a subdir without another layer of Makefile.

Reply:
The reason we have bash layer is because:
1. we need build&run UT.
2. autoconf tools difficult to generate a make file which support run UT.

Reply:
It is actually easy. This is an example: [https://github.com/Azure/sonic-swss/blob/d338bd00abf29d33fbe839c9b431ee4de5e40df7/tests/Makefile.am#L3](https://github.com/Azure/sonic-swss/blob/d338bd00abf29d33fbe839c9b431ee4de5e40df7/tests/Makefile.am#L3)